projects/cloud_feedbacks/index.md

```diff
@@ -1,463 +1,834 @@
+Cloud feedbacks are extremely variable between different climate models.
+However, it is not always clear what is the relative contribution of
+cloud types from various regions to the global mean cloud feedback and
+its inter-model spread. Researchers in this project have developed novel
+techniques to separate the contribution of different cloud types and
+have found that cloud feedbacks are not the result of a single cloud
+type but that we must consider the feedbacks from many cloud types
+including low clouds, high clouds, mid-latitude clouds and polar clouds.
+
+* WHAT ASPECTS OF CLOUD FEEDACK CAN BE CONSTRAINED WITH OBSERVATIONS WE
+HAVE TODAY?
+
+A key aspect of the project is the identification of cloud feedbacks
+where similarities are found in simulations of both current-climate
+variability and of projected climate change (so called "timescale
+invariant feedbacks"). For example, if fluctuations of clouds with
+day-to-day or season-to-season variations of temperature are similar to
+those shown over climate change time-scales, we could use observations
+from the current climate to constrain cloud feedbacks. Researchers are
+working to identify which cloud types exhibit time-scale invariance as
+well as the observations that can quantitatively constrain these
+feedbacks.
+
+* WHAT PHYSICAL PROCESSES CONTRIBUTE TO CLOUD FEEDBACK AND WHAT
+FEEDBACKS ARE CORRECT?
+
+Our confidence in any given cloud feedback is dependent on our ability
+to understand the physical processes from which the feedbacks result and
+our confidence in those processes. Researchers are working towards
+identifying the physical mechanisms of various feedbacks simulated by
+complex climate models and critiquing their realism. One technique to
+accomplish this is the application of more realistic models of cloud
+processes to the changes in the large-scale environment predicted by
+global climate models.
+
+This Cloud Feedback project is supported by the Regional and Global
+Climate Modeling
+([RGCM](http://science.energy.gov/ber/research/cesd/regional-and-global-
+modeling/)) Programs of the U.S. Department of Energy's Office of
+Science/Biological and Environmental Research
+([BER](http://science.energy.gov/ber/)).
 
 ---
 
 ### Project Participants
-
-* [Stephen A. Klein](https://pls.llnl.gov/people/staff-bios/aeed/klein-s) (Principal Investigator) 
-* [Karl Taylor](https://pcmdi.llnl.gov/staff/taylor/karlcv.html) (co-Investigator)
-* [Alex Hall](http://www.atmos.ucla.edu/csrl/) (co-Investigator, UCLA) 
-* [Peter Caldwell](https://pcmdi.llnl.gov/staff/caldwell/index.html)
-* [Timothy Myers](https://sites.google.com/site/myerstimothy/)
-* [Xin Qu](http://www.atmos.ucla.edu/csrl/) (UCLA) 
-* [Mark Zelinka](https://markdzelinka.wordpress.com/)
-* [Yunyan Zhang](https://pls.llnl.gov/people/staff-bios/aeed/zhang-y)
+### 
+* [Stephen A.
```

Comment:
These lists would be better formatted like they were previously with the name, URL, and title on one line.
```
* [Stephen A. Klein](https://pls.llnl.gov/people/staff-bios/aeed/klein-s) (Principal Investigator) 
* [Karl Taylor](https://pcmdi.llnl.gov/staff/taylor/karlcv.html) (co-Investigator) 
* [Alex Hall](http://www.atmos.ucla.edu/csrl/) (co-Investigator, UCLA) 
* [Peter Caldwell](https://pcmdi.llnl.gov/staff/caldwell/index.html) 
* [Timothy Myers](https://sites.google.com/site/myerstimothy/) 
* [Xin Qu](http://www.atmos.ucla.edu/csrl/) (UCLA) 
* [Mark Zelinka](https://markdzelinka.wordpress.com/) 
* [Yunyan Zhang](https://pls.llnl.gov/people/staff-bios/aeed/zhang-y) 
* [Xin Qu](http://www.atmos.ucla.edu/csrl/) (UCLA)
```

Reply:
Thanks, I didn't mean to do that, but I've fixed them in the latest commit.